### ISSUE TITLE
Get link very slow or impossible to get since yesterday.

### ISSUE THREAD

**Issue by cryinrain69** [HaarigerHarald/android-youtubeExtractor#199]
A few days ago, getting youtube link was very fast, but yesterday I checked, it got very slow (> 30s) or couldn't get the link.
It doesn't seem to be able to request that url (no response), while opening it in a web browser quickly.
I have updated to the latest version of the library.
Is anyone like me ?
Please help urgently.

**Comment by cryinrain69** [comment]
Anyone have the same situation as me ?
Please help me, please :-(

**Comment by trantrung22** [comment]
Seem like youtube have been throttle bandwidth for the URLs retrieved using this method. Currently [yt-dlp](https://github.com/yt-dlp/yt-dlp) can fix it

**Comment by ricky1550** [comment]
Yes i am facing the same issue. Loading of video taking more than 30 seconds. On changing the quality i am facing the same delay. Please help.

**Comment by cryinrain69** [comment]
Currently the link extracted is playing very slowly.
Anyone have a new solution ?
Please.

**Comment by trantrung22** [comment]
Currently [yt-dlp](https://github.com/yt-dlp/yt-dlp) can fix it. But it written in Python not for Android
